theme/ItaliaTheme/Components/_card.scss

```diff
@@ -67,18 +67,7 @@
       @include rem-size(font-size, 18);
       @include rem-size(line-height, 24);
       font-family: 'Titillium Web' !important;
-    }
-  }
-
-  .card-image {
-    flex-basis: 20%;
-    @include rem-size(max-width, 111.18);
-    max-height: -webkit-fill-available;
-    max-width: 111px;
-    margin: auto;
-
-    img {
-      width: -webkit-fill-available;
+      width: 60%;
```

Comment:
Così a occhio mi sembra che questa modifica imposti il 60% sempre per il testo, anche quando un'immagine non c'è.

Reply:
Ora il testo ha width diverso in base alla presenza o meno dell'immagine :)